Sprint-3/implement/get-card-value.js

```diff
@@ -51,16 +51,11 @@ let validFaceCards = ["J", "Q", "K", "A"];
         throw new Error("Invalid card rank or card suit");
     }
  
-//Handling when card inputs are 2-9
-    if (rank >= "2" && rank <= "9") {
+//Handling when card inputs are 2-10
+    if (!isNaN(rank) && rank.length<=2 && Number(rank) >= "2" && Number(rank) <= "10") {
```

Comment:
Do you want to accept rank such as "07" in your design?

When we treat strings as number, we have to consider a lot of possibilities.
It is easier to
- First, ensure the string matches our expected pattern (e.g., one of 2", "3", ...., "10")
- Next, convert the string to its equivalent numerical value.

Your previous code is actually quite solid.
```
//Handling when card input is 10
    if (rank === "10") {
        return 10;
    }    
```